src/firestore/firestore.ts

```diff
@@ -122,10 +118,12 @@ export class AngularFirestore {
     });
 
     this.persistenceEnabled$ = zone.runOutsideAngular(() =>
-        shouldEnablePersistence ? from(this.firestore.enablePersistence().then(() => true, () => false))
-                                : of(false)
+        shouldEnablePersistence ? Observable.from(this.firestore.enablePersistence().then(() => true, () => false))
```

Comment:
we should import `of` and `from` rather than using `Observable.from` and `Observable.of`